app/views/rmt_view.py

```diff
@@ -21,6 +21,8 @@ def get_emp_info(name, on_false={}, on_missing='Unspecified'):
 
   mob_number = device_numbers[0] if len(device_numbers) > 0 else ''
 
+  employee_info['address'] = map_full_address_details(employee_info)
```

Comment:
Ditto, as with the above comment on input mutation.

Reply:
Changed too